org.scala-ide.sdt.core.tests/src/org/scalaide/core/testsetup/Bdd.scala

```diff
@@ -0,0 +1,55 @@
+package org.scalaide.core.testsetup
+
+import java.io.File
+
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.resources.IncrementalProjectBuilder
+import org.eclipse.core.runtime.NullProgressMonitor
+import org.eclipse.jdt.core.IJavaProject
+import org.junit.Assert
+import org.scalaide.core.IScalaProject
+import org.scalaide.util.eclipse.EclipseUtils
+
+import SDTTestUtils.enableAutoBuild
+import SDTTestUtils.getErrorMessages
+import SDTTestUtils.sourceWorkspaceLoc
+import SDTTestUtils.workspace
+
+/**
+ * Behavioral Driven Development methods.
+ */
+trait Bdd {
```

Comment:
This name seems odd to me. Why not putting its member directly in `SDTTestUtils`?